config/global.ini.php

```diff
@@ -521,6 +521,9 @@
 ; Default is 0 (i.e., bust frames on the Settings forms).
 enable_framed_settings = 0
 
+; Set to 1 to force send request from ssl
+enable_ssl_request = 1
```

Comment:
Honestly I think adding a new option in addition to `force_ssl` will just add more confusion (at least without detailed documentation explaining the difference and when to use which one). 
(Always use the new option, unless your outgoing SSL-setup is completely broken and you are okay with everything being insecure, use the old option when your Matomo instance uses HTTPS (which also should be always unless you are okay with everything being insecure or you are running Matomo as an onion service))